roles/commcare_analytics/templates/superset/superset_config.py.j2

```diff
@@ -50,9 +50,12 @@ FEATURE_FLAGS = {
     "DASHBOARD_NATIVE_FILTERS": True,
     "DASHBOARD_NATIVE_FILTERS_SET": True,
     "DASHBOARD_FILTERS_EXPERIMENTAL": True,
+    "ENABLE_JAVASCRIPT_CONTROLS": True,
+    "ENABLE_TEMPLATE_PROCESSING": True
```

Comment:
What are these for?

Reply:
These FFs are enabled in prod at the moment. This is just so we don't disable them when deploying again. It's so easy to enable FFs in prod without having to deploy, it can become too easy to override what's running in prod.

Reply:
Absolutely. Sounds like we need to document and share a policy to only manage feature flags in this file, like what we do with other environments.

Reply:
> a policy to only manage feature flags in this file

Meaning that we'd have to deploy to enable a FF? Or making a PR with the FF enabled (like this one) whenever we update it on prod's `superset_config.py`?

Reply:
> Meaning that we'd have to deploy to enable a FF?

At the time I meant that we change the FF in this repo and then deploy. That way we know that this repo is always a true representation of what is live, so there is no risk with deploys breaking things unexpectedly.

However, back when I made that suggestion, I believed that this repo is used for deploys. Now I know that that has never been true. The code here cannot do a deploy. It is incomplete, and untested.